### PR TITLE
Feature/url export

### PR DESCRIPTION
**Pull request by heinrichmartin**
I was not able to build or test the changes. Please review and test.

Also, I noted that several files use DOS line endings, no line feed before EOF, and encoding bytes, which git does not like. Is this on purpose?

### CODE REVIEW

**Comment by danielsteiner** [comment]
I took a quick look at the code and it looks good so far, I will try to deploy it to my phone in a few hours.

File: GeoClient/GeoClient/Services/Registration/RegistrationService.cs [hunk 1]
```diff
@@ -85,6 +86,21 @@ public RegistrationInfo GetRegistrationInfo()
             return _cachedRegistrationInfo;
         }
 
+        public bool OpenUri()
+        {
+            bool result = false;
+            if (IsRegistered())
+            {
+                Device.OpenUri(new Uri(_cachedRegistrationInfo.Url));
```
Comment (robo-w) [inline GeoClient/GeoClient/Services/Registration/RegistrationService.cs]: The service should not know something about UI concerns. I will move this to the UI class

File: GeoClient/GeoClient/Views/AboutPage.xaml [hunk 2]
```diff
@@ -92,6 +92,22 @@
...
                 </Label>
+
+                <Label>
+                    <Label.FormattedText>
+                        <FormattedString>
+                            <Span Text="Url: " FontAttributes="Bold" />
+                            <Span
+                                Text=""
+                                x:Name="ContentUnitUrl"
+                                TextDecorations="Underline" />
```
Comment (robo-w) [inline GeoClient/GeoClient/Views/AboutPage.xaml]: typo: closing slash

File: GeoClient/GeoClient/Services/Registration/RegistrationService.cs (hunk 1 above)
Comment (heinrichmartin) [reply GeoClient/GeoClient/Services/Registration/RegistrationService.cs]: Yes, please. I don't have the build env and chose the least risk ...

**Pull request merged**